go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/chain-selectors v1.0.27
 	github.com/smartcontractkit/chainlink-automation v0.8.0
-	github.com/smartcontractkit/chainlink-ccip v0.0.0-20241014104242-9227e5c976a7
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20241016204615-8a07a53b044c
```

Comment:
TODO: point this to main branch once https://github.com/smartcontractkit/chainlink-ccip/pull/240 is merged.

Reply:
done